### ISSUE TITLE
Issue on page /resources/probability.html

### ISSUE THREAD

**Issue by henrymjohnson** [mdekstrand/cs533-web#3]
The symbol used for the function that maps the expected value to the probability doesn't display properly. I've attached a screenshot and the following text is copied from the course site (the A does properly show up as a subscript). This happens in both Safari and Chrome. I didn't check Firefox.

𝕀𝐴:𝐸→{0,1}

<img width="1584" alt="image" src="https://user-images.githubusercontent.com/13862226/133297501-9e3a32e7-476e-4c75-9947-43560558e50f.png">

**Comment by mdekstrand** [comment]
I'm not seeing the problem yet - it shows that the indicator I_A is a function from E to the set {0,1}. Could you describe what you're expecting to see?

**Comment by henrymjohnson** [comment]
I'm expecting to see a symbol (for the Indicator Function) that's not an empty box followed by the subscript _A_. I was thinking that my version of Safari just didn't include the character for whatever symbol is used for the function here. So I opened Chrome to see if it displayed how I'd expect there

<img width="780" alt="image" src="https://user-images.githubusercontent.com/13862226/133312868-cb1df114-59d7-4fcf-93d6-3cc2ef4f6564.png">

**Comment by henrymjohnson** [comment]
Oh, maybe it's a bubble-type letter for the letter 'I' 😅

**Comment by henrymjohnson** [comment]
I'll close the issue. Thanks for the clarification

**Comment by mdekstrand** [comment]
Ok, I see. That is supposed to be a double-struck I (`\mathbb{I}`). On my computer, it renders with the serif, making it clearer. 

It does, however, look like that is an empty box, not a double-struck I - perhaps the MathJax fonts are not fully working in your browser. I'll check on my Mac later today.

**Comment by henrymjohnson** [comment]
That was exactly my thought, too. So I checked Chrome (and used Chrome for the screenshots). It rendered the same. I thought that fonts were browser-specific and didn't really think about browsers being OS-specific haha. After you said that it's the double-struck I, I noticed it was more rectangular than the normal missing symbol box. So it may just be that the double-struck I symbol displayed for the browsers (Safari and Chrome) on OS X is ... a poor choice 🤷‍♂️

**Comment by mdekstrand** [comment]
I have pushed a fix that I believe works, should be live shortly.

Turns out that MathJax (2.7, at least) tries to first use local fonts if they are available, and falls back to webfonts if they are not. macOS includes the STIX fonts, so MathJax tries to use them; however, they do not seem to contain the proper double-struck I symbol, because that's definitely a blank box.

The fix I pushed disables use of the STIX fonts, so macOS browsers will always use the web-based MathJax TeX fonts. This corrected display in my local testing.
